charts/karpenter/templates/webhook/deployment.yaml

```diff
@@ -46,8 +46,8 @@ spec:
               containerPort: 8081
           livenessProbe:
             httpGet:
-              path: /healthz
-              port: 8081
+              scheme: HTTPS
```

Comment:
A couple of quick questions that may just be my lack of understanding of liveness probes:

- Do we still need a path here? 
- Does putting the port to the same port as the webhook container mean it only pings healthy when that container is healthy?
- Do we still need the health-probe port at 8081?

Reply:
> Do we still need a path here?
Knative pkg sets up probe handlers here: https://github.com/knative/pkg/blob/b558677ab03404ed118ba3e179a7438fe2d8509a/network/handlers/drain.go#L83 that do not require a path and share the port with the webhook. It's actually pretty cool how it works. I just missed this the first time around and it was a good opportunity to simplify and remove our custom webhook.

> Does putting the port to the same port as the webhook container mean it only pings healthy when that container is healthy?
Yeah more or less. See the code above.

>Do we still need the health-probe port at 8081?
Good catch! No! Removed.